backend/app/main.py

```
@@ -0,0 +1,30 @@
+from typing import Union
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)